src/status_im/transport/message/transit.cljs

```diff
@@ -103,7 +96,9 @@
                               "c5" (fn [message-ids]
                                      (v1.protocol/MessagesSeen. message-ids))
                               "c6" (fn [[name profile-image address fcm-token]]
-                                     (v1.contact/ContactUpdate. name profile-image address fcm-token))}})) ; removed group chat handlers for https://github.com/status-im/status-react/issues/4506
+                                     (v1.contact/ContactUpdate. name profile-image address fcm-token))
+                              "g5" (fn [[chat-id chat-name admin participants signature message]]
```

Comment:
what about g3 group leave for instance ?

Reply:
I think g3 was taken already, I had removed it, unlikely to be a problem but I just kept counting :)